src/libprojectM/ProjectM.cpp

```diff
@@ -585,6 +534,7 @@ void ProjectM::projectM_init(int gx, int gy, int fps, int texsize, int width, in
     m_timeKeeper->StartPreset();
     assert(pcm());
 
+    // ToDo: Calculate the real FPS instead
```

Comment:
CLion [can display a list with all "todo" items](https://user-images.githubusercontent.com/5038276/165306668-4216a1d2-d012-4d73-8304-a90aeb3405c5.png) in the source, so that's actually how I track things I still need to do. It's currently set to a static value via configuration.

Reply:
There's a new Projects beta running now, need to look into it what it can provide. The previous version was just able to pin issues and PRs plus "notes" to the board, but didn't allow for subtasks and other cards linked to certain work items like branches or issues.